### PR TITLE
add instructions to connect with mosquitto and MQTT explorer

### PR DESCRIPTION
**Pull request by rido-min**
This PR Includes  instructions to connect to the broker with existing clients, it adds:
-  the baltimoreCA public key
- a bash script to subscribe with mosquitto_sub cli
- screenshots to help configuring MQTT Explorer
- A .md file with instructions

thanks to @digimaun for the CLI scripts

### CODE REVIEW

File: references/mosquitto_sub.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```
Comment (prashmo) [inline references/mosquitto_sub.sh:1]: So this is specific to Linux?

File: references/mosquitto_sub.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+hub=$1
+dev=$2
+
+az iot hub device-identity show -n $hub -d $dev --query properties.deviceId  2>/dev/null || \
+    (echo "creating device $dev" && \
+    az iot hub device-identity create -n $hub -d $dev)
+
+mqttCreds=$(az iot hub device-identity generate-mqtt-credentials -n $hub -d $dev --du 3600 --only-show-errors)
```
Comment (prashmo) [inline references/mosquitto_sub.sh:10]: consider using a variable for `3600` some thing like expiry_duration

File: references/mosquitto_sub.sh (hunk 1 above)
Comment (rido-min) [reply references/mosquitto_sub.sh:1]: yes, (well with WSL this works also on Windows).

File: references/mosquitto_sub.sh (hunk 2 above)
Comment (rido-min) [reply references/mosquitto_sub.sh:10]: .. I'd prefer to remove it and use the default value to keep things simple

File: references/using-existing-mqtt-clients.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+- Encryption (tls): enabled
+- Protocol: mqtt://
+- Host: myiothubwithbroker.azure-devices.net
+- Port: 8883
+- Username: <copy/paste the user name obtained with az iot hub device-identity generate-mqtt-credentials >
+- Password: <copy/paste the user password obtained with az iot hub device-identity generate-mqtt-credentials >
+- Advanced/ClientId: <your device id>
+- Advanced/Certificate/Server Certificate (CA): path to BaltimoreCA.pem
+
+![MQTT Explorer Options](mqttex_options.png)
```
Comment (prashmo) [inline references/using-existing-mqtt-clients.md:76]: cant see the image, will share comments offline, post commit.

**Review by prashmo — APPROVED** [review]

File: references/using-existing-mqtt-clients.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+[MQTT Explorer](https://mqtt-explorer.com/) is a popular GUI tool you can use to publish/subscribe to any MQTT compliant broker. Once you installed the tool you need to configure the connection with the next options:
+- Name: The name of your connection
+- Validate Certificate: enabled
+- Encryption (tls): enabled
+- Protocol: mqtt://
+- Host: myiothubwithbroker.azure-devices.net
+- Port: 8883
+- Username: <copy/paste the user name obtained with az iot hub device-identity generate-mqtt-credentials >
+- Password: <copy/paste the user password obtained with az iot hub device-identity generate-mqtt-credentials >
+- Advanced/ClientId: <your device id>
```
Comment (prashmo) [inline references/using-existing-mqtt-clients.md]: As discussed, please fix to make it visible in mark down

**Pull request merged**